assets/src/scripts/monitoring-location/components/hydrograph/download-links.js

```diff
@@ -53,19 +21,22 @@ const createUrlForDownloadLinks = function(currentIVDateRange, queryInformation,
 * @param {store} store - The Redux store, in the form of a JavaScript object
 * @param {String} siteno- a USGS numerical identifier for a specific monitoring location
 */
-/*
 export const renderDownloadLinks = function(elem, store, siteno) {
+    const monitoringLocations = [siteno]; // The method that processes the service URL expects an array of locations
+
     elem.call(link(store, (elem, {
-        currentIVDateRange,
-        parameterCode,
-        showIVTimeSeries,
-        queryInformation,
-        medianData,
-        anyVisibleGroundwaterLevels,
-        requestTimeRange
+        currentTimeRange,
+        priorYearTimeRange,
+        inputs,
+        medianStatisticsData,
```

Comment:
I wonder if we should make selectors that return true if there is any data of a particular kind. Seems like I have checked this elsewhere too.

Reply:
I debated about this. To me, it seemed like an extra computing step, since in both cases the function has to check the state. But I'm okay with the idea, if you think it works better.

Reply:
I don't feel really strongly about it. Maybe once this code is merged in, I'll do a check to see if I think it's worth it. I think this is fine for now.